com.sublimehq.SublimeText.yaml

```diff
@@ -0,0 +1,88 @@
+app-id: com.sublimehq.SublimeText
+command: subl
+
+runtime: org.freedesktop.Sdk
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --device=dri
+  - --socket=x11
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --talk-name=org.gnome.SettingsDaemon
+  - --filesystem=host
+  - --env=PATH=/app/utils/bin:/app/sublime_merge/bin:/app/bin:/usr/bin
+  - --env=PYTHONPATH=/app/utils/lib/python3.8/site-packages
```

Comment:
this is the wrong python version

Reply:
Well, Sublime Text ships with 3.8, and that's what all the plugins run on (unless they are old, in which case there is also a 3.3 version included).